clean/plaquemines_so_vine_booking.py

```diff
@@ -0,0 +1,33 @@
+import requests
+from bs4 import BeautifulSoup
+import pandas as pd
+from lib.columns import clean_column_names
+from lib.path import data_file_path
+
+
+url = 'http://www.plaquemines.lavns.org/roster.aspx'
+r = requests.get(url)
+soup = BeautifulSoup(r.text, 'html.parser')
+
+inmates_dict = []
+for table_row in soup.select("tr"):
+    cells = table_row.findAll('td', {'align': 'left', 'valign': 'middle'})
+    if len(cells) > 0:
+        name = cells[0].text
+        dob = cells[1].text
+        race = cells[2].text
+        gender = cells[3].text
+        arrest_date = cells[4].text
+        inmate = {'name': name, 'birth_date': dob, 'race': race, 'gender': gender, 'arrest_date': arrest_date}
+        inmates_dict.append(inmate)
+
+
+def clean():
+    df = pd.DataFrame.from_dict(inmates_dict)\
```

Comment:
Not sure how this is different but for constructing a dataframe from a list of dicts I usually use `DataFrame.from_records`.